test/integration/targets/ec2_ami/tasks/main.yml

```diff
@@ -295,7 +319,6 @@
         security_token: '{{security_token}}'
         state: present
         image_id: '{{ result.image_id }}'
-        name: '{{ ec2_ami_name }}_ami'
```

Comment:
This task now tests the thing https://github.com/ansible/ansible/pull/38514 fixed.